src/app/public/modules/country-field/country-field.component.html

```diff
@@ -0,0 +1,59 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <span
+    class="sky-country-field-container"
+    [ngClass]="{
+      'sky-country-field-disabled': disabled
+    }"
+  >
+    <form
```

Comment:
Why does the component have its own `form` element? It's bad practice to nest `form` elements, so a consumer could unwillingly put this input inside their own `form` element. Is this absolutely necessary?

Reply:
Nope. I was able to remove this. I thought you had to have it in order to use the reactive form but I got it to work without it.